scripts/bootloader/generate_zip.py

```diff
@@ -36,6 +36,8 @@ def parse_args():
                         file.''')
     parser.add_argument('--name', required=False, help='Optional name to display to the user to help identify the '
                         'purpose of this update')
+    parser.add_argument('--format-version', required=False, type=int, default=0,
```

Comment:
I think we should default to `1`, which is the current one.

Reply:
Discussed offline. I will set default to `1` here